d2l-localize-behavior.js

```diff
@@ -1,180 +1,138 @@
 import '@polymer/polymer/polymer-legacy.js';
-import { AppLocalizeBehavior } from '@polymer/app-localize-behavior/app-localize-behavior.js';
-import d2lIntl from 'd2l-intl';
+import {
+	addListener, formatDateTime, formatDate, formatFileSize, formatNumber,
+	formatTime, getDocumentLanguageFallback, getDocumentLanguage, getTimezone,
+	localize, parseDate, parseNumber, parseTime, removeListener
+} from '@brightspace-ui/core/helpers/localization.js';
+
 window.D2L = window.D2L || {};
 window.D2L.PolymerBehaviors = window.D2L.PolymerBehaviors || {};
 
-/** @polymerBehavior D2L.PolymerBehaviors.LocalizeBehaviorImpl */
-D2L.PolymerBehaviors.LocalizeBehaviorImpl = {
+/** @polymerBehavior */
+D2L.PolymerBehaviors.LocalizeBehavior = {
 	properties: {
 		formatDateTime: {
 			type: Function,
-			computed: '_computeFormatDateTime(language, __overrides, __timezone)'
+			computed: '_computeFormatDateTime(language)'
```

Comment:
This means changes to the overrides and timezone no longer trigger a re-render of things. This wasn't a real-world use case anyway so I was OK with losing that functionality.